src/main/java/edu/tamu/scholars/middleware/discovery/component/solr/SolrIndexer.java

```diff
@@ -69,17 +69,17 @@ public void init() {
                 try {
                     SchemaRequest.AddField addFieldRequest = new SchemaRequest.AddField(fieldAttributes);
                     addFieldRequest.process(solrClient, collectionName);
-                } catch (Exception e) {
-                    logger.debug("Failed to add field", e);
-                }
 
-                if (fieldType.copyTo().length > 0) {
-                    try {
-                        SchemaRequest.AddCopyField addCopyFieldRequest = new SchemaRequest.AddCopyField(name, Arrays.asList(fieldType.copyTo()));
-                        addCopyFieldRequest.process(solrClient, collectionName);
-                    } catch (Exception e) {
-                        logger.debug("Failed to add copy field", e);
+                    if (fieldType.copyTo().length > 0) {
+                        try {
+                            SchemaRequest.AddCopyField addCopyFieldRequest = new SchemaRequest.AddCopyField(name, Arrays.asList(fieldType.copyTo()));
+                            addCopyFieldRequest.process(solrClient, collectionName);
+                        } catch (Exception e) {
+                            logger.error("Failed to add copy field", e);
+                        }
                     }
+                } catch (Exception e) {
+                    logger.debug("Failed to add field", e);
```

Comment:
If the copy field is `logger.error` should not this be `logger.error` rather than `logger.debug`?

Reply:
Not if schematization is unintentionally left on in the configuration. Currently Solr responses are not considered and if it was successful or not. On restart a bunch of these logs on each instance would be wasteful.

Reply:
Since debugging can be turned on. I prefer this to be a debug as we have shown deterministic behavior of the indexing process.